docs/about/release-notes.rst

```diff
@@ -35,33 +35,52 @@ Release Notes
    and Jan-Lukas Wynen :sup:`a`
 
 
-v22.11.1 (January 2023)
+v23.xy.0 (January 2023)
 -----------------------
 
+Features
+~~~~~~~~
+
+* Added support for arbitrary unit, degrees Celsius and other special units `#2931 <https://github.com/scipp/scipp/pull/2931>`_.
+
+Breaking changes
+~~~~~~~~~~~~~~~~
+* The HDF5 format changed to accommodate more units, old files can still be loaded but new ones cannot be loaded with an old version of Scipp `#2931 <https://github.com/scipp/scipp/pull/2931>`_.
```

Comment:
Not even if they do not contain the flag? I remember you only stored nonzero powers, or was that changed in the final implementation?

Reply:
It could in principle work. But the loader will reject the file because the version was increased. I could revert this but then loading would silently load wrong units if, e.g. `e-flag` is set. The old loader simply ignores it.